samplers/jaegerremote/logger.go

```diff
@@ -0,0 +1,40 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jaegerremote // import "go.opentelemetry.io/contrib/samplers/jaegerremote"
+
+import "github.com/go-logr/logr"
+
+// NullLogger is implementation of the Logger interface that is no-op
+var NullLoggger = &nullLogger{}
```

Comment:
Please use the [`Discard`](https://pkg.go.dev/github.com/go-logr/logr#Discard) logger instead of adding this here.

Reply:
done.